msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -475,6 +580,27 @@ The object is an array (maximum of 16 elements) with all elements of the type `o
 |`searchUrl`|String|2048 characters||The https:// URL to direct a user's search queries.|
 |`context`|Array of enum|8||The set of `contextItem` scopes to which a tab belongs. <br>Default values: `personalTab`, `channelTab`, `privateChatTab`, `meetingChatTab`, `meetingDetailsTab`, `meetingSidePanel`, `meetingStage`, `teamLevelApp`|
 |`supportedPlatform`|Array of enum|3||The set of `supportedPlatform` scopes to which a tab belongs. <br>Default values: `desktop`, `mobile`, `teamsMeetingDevices`|
+|`requirementSet`|Object|||Runtime requirements for the tab to function properly in the Microsoft 365 host application. If one or more of the requirements aren't supported by the runtime host, the host won't load the tab.|
```

Comment:
@erikadoyle, how do we reduce the content redundancy here? Just thinking out loud, can we skip directly to `staticTabs.requirementSet.hostMustSupportFunctionalities` and refer to one of the code snippets? Let me know if you have any ideas. Applicable to both bots and message extensions down the line.

Reply:
It's a fair point, but should we do it? I'm thinking refactoring makes it easier for us to maintain, but puts burden on the customer (who then must jump around the doc and potentially lose context of their tab/bot/composeExtension documentation).

Reply:
@erikadoyle, I agree with you that refactoring is advantageous here. If we are to help the reader jump around less, we'd need to double down here - keep the original content (or trim it) and add multiple H3s here.

If we want to eliminate content redundancy, we can go with either skipping directly to `capability.requirementSet.hostMustSupportFunctionalities` or add multiple H3s here. @v-deabira, @v-npaladugu, thoughts?

Reply:
Thanks--commited your inline suggestions